docs/GETSTARTED.md

```diff
@@ -84,7 +84,7 @@ Another way is to reach out to our community leaders. Instructions on when and h
 ## Link Archive
 Here you can find some useful links:
 - **[GitHub project](https://github.com/orgs/deven-org/projects/3/views/31?pane=info)** - This is where we are managing our work. Basically it is better view of all the issues in the repository. Here you can find all our boards, our backlogs and our roadmaps. The documentation of all this can be found in the side-panel.
-- **[Figma file](https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?type=design&node-id=1134-44297&mode=design&t=BFbmHTvPYeKQBmHt-0)** - This is the main Figma file. Here you can find all the component designs and all the design tokens via the Tokens Studio plugin.
+- **[Figma file](https://www.figma.com/file/kG44mSWhEp2VcMvMqbJrRY/B01LER-%5Bv0.2--alpha-release%5D?type=design&node-id=1134%3A44297&mode=design&t=ZatzD7omF9pxalvP-1)** - This is the main Figma file. Here you can find all the component designs and all the design tokens via the Tokens Studio plugin.
```

Comment:
This change is not right. Please leave the link as it is. We will not continue working in the new 0.2 release file, but in the previous one.